js/timepicker-ng.js

```diff
@@ -1051,7 +1051,7 @@
       // show/hide approach taken by datepicker
       this.$widget.appendTo(this.appendWidgetTo);
       var self = this;
-      $(document).on('mousedown.timepicker, touchend.timepicker', function (e) {
```

Comment:
The separator for event names is SPACE, not comma.
The original code will bind handler on event "mousedown.timepicker**,**" and "touchend.timepicker"